### ISSUE TITLE
Better tools for welcoming new users

### ISSUE THREAD

**Issue by bhousel** [osmlab/osm-planning#13]
from https://wiki.openstreetmap.org/wiki/Top_Ten_Tasks#Better_tools_for_welcoming_new_users

### Better tools for welcoming new users
* Who can you talk to: [dev mailing list](https://lists.openstreetmap.org/listinfo/dev) or [EWG](http://www.osmfoundation.org/wiki/Engineering_Working_Group)
* What's being done already: see list below
* When a user registers to OpenStreetMap, they see a [welcome page](https://www.openstreetmap.org/user/Glassman/diary/36237) outlining what OSM is and where to get help. The contents are quite generic, with many channels (especially local ones) absent, and lack a human touch.

Many mappers are putting an admirable effort into filling that gap by welcoming new users from their area to the OSM community, as this kind of local human contact is often considered the best way of helping beginners feel at home in OSM. These messages tend to be written in the local language. They can involve giving feedback to new users' changes (especially those using the `review_requested=yes` changeset tag), but also just sending a friendly welcome mail, offering a point of contact to the local community, sharing experience with local data sources and imagery, and/or inviting users to local [user groups](https://wiki.openstreetmap.org/wiki/User_group), regional mailing lists or forums.

Developers can help encourage this kind of grassroots community building by providing tools that make it easier to notice and contact new users joining a local community.

Existing tools include:

* [OSM Community Index](https://github.com/osmlab/osm-community-index), currently integrated into iD's editor changeset upload dialog (see #5)
* [OSMCha](https://wiki.openstreetmap.org/wiki/OSMCha) (focus on changeset reviews)
* [OOOC](http://resultmaps.neis-one.org/oooc) (Overview of OpenStreetMap Contributors aka Who's around me?
* [welcoming workflow by the Belgian community](https://github.com/osmbe/osm-welcome-belgium)
* various [feeds](https://wiki.openstreetmap.org/wiki/Feeds) of editing activity
* the "home location" feature on osm.org and other ways to find [nearby users](https://wiki.openstreetmap.org/wiki/Nearby_users)

---
_This issue comes from the [OSM Top Ten Tasks](https://wiki.openstreetmap.org/wiki/Top_Ten_Tasks) page_

**Comment by bhousel** [comment]
Consider CRM system like HelpScout / Salesforce

**Comment by mcawley** [comment]
Embed community / user map on the OpenStreetMap US site

**Comment by mcawley** [comment]
Discussion in May 22 Hackathon about creating a new welcome tool/data table.  This crowd source tool would be used to welcome new users to OSM. Anyone can participate and welcome new users. Suggestions on what you might want to say and links to common resources are provided.
Functionality: 
Tool to make welcoming new OSMers easier
Sort by geography edited
Sort by hash tag
Tracks when a welcome message is sent and who sent it
Similar to Belgian community site used to track new editors and efforts to welcome each person.

**Comment by mcawley** [comment]
@joostschouppe we have been talking about the welcome page for the Belgium community - it's great! Any advice or input in this discussion is welcome! Have you seen any changes in new mapper statistics since you implemented a welcome panel? Thanks!

**Comment by joostschouppe** [comment]
@mcawley yes, I did study the impact of the welcoming message. We have a quasi-experimental setting, as we are not always sending the messages. I didn't analyse it really really good, but from what I could see it had basically zero impact on the statistics. But it does have an impact on registrations to our channels; though probably not as much as the new osm-community-index integration in iD.
Many people have said that it feels like it is only when you really personalise the message, that you feel like you are having an impact. So I would prefer a fully automated approach + a fully personalised approach, and not this hybid of the two.

BTW, the Italian community [rebuilt the tool](http://welcome.openstreetmap.it) in some other language, but I can't find the repo right now.

**Comment by mcawley** [comment]
Thank you @joostschouppe  - that helps! We were thinking of trying the fully personalized approach - I will keep you in the loop as things develop! Also, thanks for the link to the Italian tool.

**Comment by sabas** [comment]
Link to the source code for my tool https://github.com/osmItalia/welcome-tool
